addons/website_form/static/tests/tours/website_form_editor.js

```diff
@@ -91,7 +91,35 @@ odoo.define('website_form_editor.tour', function (require) {
             trigger: '#oe_snippets .oe_snippet:has(.s_website_form) .oe_snippet_thumbnail',
             run: 'drag_and_drop #wrap',
         }, {
-            content: "Check dropped snippet and select it",
+            content: "Select form by clicking on an input field",
+            extra_trigger: '.s_website_form_field',
+            trigger: 'section.s_website_form input',
+        }, {
+            content: "Verify that the form editor appeared",
+            trigger: '.o_we_customize_panel .snippet-option-WebsiteFormEditor',
+        }, {
```

Comment:
This is missing the empty run method, to indicate this is just a check.
Without it, the test-tour is actually clicking on the element.